src/test/java/io/crate/testing/BuilderTest.java

```diff
@@ -72,6 +74,37 @@ public void testBuilder() throws Throwable {
     }
 
     @Test
+    public void testBuilderKeepWorkingDir() throws Throwable {
+        CrateTestServer testServer = CrateTestServer.fromVersion("0.52.0")
+                .clusterName("mycluster")
+                .httpPort(12345)
+                .transportPort(55555)
+                .keepWorkingDir(true)
+                .build();
+
+        testServer.before();
+        assertThat(testServer.crateWorkingDir().exists(), is(true));
+
+        testServer.after();
+        assertThat(testServer.crateWorkingDir().exists(), is(true));
+    }
+
+    @Test
+    public void testBuilderDoNotKeepWorkingDir() throws Throwable {
+        CrateTestServer testServer = CrateTestServer.fromVersion("0.52.0")
```

Comment:
do you need all those options? if not remove them, since it has nothing to do with the assertion